src/styles/base.css

```diff
@@ -873,3 +873,19 @@ a.card:hover {
 #pluginPage #grid-box .content .main td {
   padding: 1em;
 }
+
+#pluginPage .alert {
```

Comment:
Can you wrap the component in a class or id so these css can be tied to just the component? Don't want to have all alerts on a plugin page changed, just the governance ones.